### PR TITLE
[Docs]: Who Can Use This Feature in meshmap and cloud

### LINKED ISSUE #89: Create a "Who Can Use This Feature" layout/block/shortcode
#### Current Behavior
Some features of Cloud and MeshMap are available to all users, while other features are available only at specific subscription levels.

#### Desired Behavior
Use frontmatter designations to drive a notice at the top of the page for which subscription plan this feature is applicable.

#### Implementation
Each docs page should be capable of having frontmatter like:

```
plan: free, team, enterprise
```

#### Acceptance Tests
Every page has a callout for subscription applicability.

#### Mockups
[Examples from Figma docs](https://help.figma.com/hc/en-us/articles/360039825314-Guide-to-comments-in-Figma) - 
Screenshot 2023-11-21 at 4 05 39 PM

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

### PR DESCRIPTION
**Pull request by SAHU-01**
**Notes for Reviewers**

This PR fixes #89

1. Adds a YML datafile comprising information about the features available under each subscription plan
2. Adds front-matter on top of each feature page in documentation 

image

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ashparshp** [comment]
@SAHU-01, Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by leecalcote** [comment]
@sudhanshutech @Yashsharma1911 @vishalvivekm please confirm that this data will be centrally tracked in spreadsheet and programmatically published / refreshed.

File: assets/scss/_styles_project.scss [hunk 1]
```diff
@@ -381,6 +381,49 @@ a:not([href]):not([class]):hover {
...
+.plan-icon {
+  display: inline-block;
+}
+
+.highlight{
+  color: #00b39f; 
+  font-weight: bold; 
+  margin-left:10px
+}
+
```
Comment (leecalcote) [inline assets/scss/_styles_project.scss:426]: I wonder if this subscription function might benefit from a new `_styles_<something>.scss` in the hopes of keeping CSS files shorter, than longer.

**Review by marblom007 — COMMENTED** [review]
In general, this is a nice feature of the docs to have, so that it's readily evident what features and functions are available in which subscription plan.

I wonder, though, am I seeing right list item icon? This is a Discourse icon. Is there a special reason for this? What does it mean?

Screenshot 2024-08-20 at 5 02 46 PM

**Comment by SAHU-01** [comment]
> In general, this is a nice feature of the docs to have, so that it's readily evident what features and functions are available in which subscription plan.
> 
> I wonder, though, am I seeing right list item icon? This is a Discourse icon. Is there a special reason for this? What does it mean?
> 
> Screenshot 2024-08-20 at 5 02 46 PM

Yes, regarding that icon, that is the disclosure icon, the most suitable icon i could find, I just added that and waited for a feedback, I'll be changing it, I've received an icon svg that matches the subscription usecase.
Thanks for pointing that out though!

File: assets/scss/_styles_project.scss (hunk 1 above)
Comment (SAHU-01) [reply assets/scss/_styles_project.scss:426]: Sure, I'll do that!

**Pull request closed without merging**